build.sbt

```diff
@@ -22,7 +22,7 @@ ThisBuild / javacOptions ++= List(
   "-Xlint:deprecation",
 )
 
-resolvers += Resolver.bintrayRepo("akka", "maven")
+ThisBuild / resolvers += Resolver.bintrayRepo("akka", "maven")
```

Comment:
@raboof this is why it was not resolving the dependencies.

Reply:
Aah, I must've already had them in my ~/.ivy2 so I didn't notice